lib/lib-table-query/src/main/resources/table-query-parser.jj

```diff
@@ -234,14 +234,28 @@ TOKEN [IGNORE_CASE]: {
   >
 }
 
+/* Synapse functions
+TOKEN [IGNORE_CASE]: {
+  < synapse_function_name:
+   "CURRENT_USER"

```

Comment:
You might want to add the documentation in the table examples, see SQLSpecification2.csv in lib-javadoc